volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
```

Comment:
zactly

On 24 May 2016, at 9:46, Madhav Puri wrote:

>> @@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
>>  	r := mux.NewRouter()
>>
>>  	postRouter := map[string]func(http.ResponseWriter, *http.Request){
>> -		"/request":      d.handleRequest,
>> -		"/create":       d.handleCreate,
>> -		"/copy":         d.handleCopy,
>> -		"/mount":        d.handleMount,
>> -		"/mount-report": d.handleMountReport,
>> -		"/unmount":      d.handleUnmount,
>> -		"/remove":       d.handleRemove,
>> +		"/global":                           d.handleGlobalUpload,
>> +		"/volumes/create":                   d.handleCreate,
>
> yes, sgtm !
>
> I also think until 1.0 we can change APIs in a backward incompatible 
> way, so we will have some leeway to rethink some of this.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/255/files/12004ac3ec0c06ee7937e32845ee33e7c45a120f#r64429715